src/please.go

```diff
@@ -416,6 +416,7 @@ var opts struct {
 		Changes struct {
 			Since            string `short:"s" long:"since" default:"origin/master" description:"Revision to compare against"`
 			IncludeDependees string `long:"include_dependees" default:"none" choice:"none" choice:"direct" choice:"transitive" description:"Deprecated: use level 1 for direct and -1 for transitive. Include direct or transitive dependees of changed targets."`
+			IncludePlugins   bool   `long:"include_plugins" description:"Include changed targets that belong to plugin subrepos."`
```

Comment:
Maybe this should just be all subrepos? I feel like I would probably want the same behaviour most of the time from non-plugin repos as well (e.g. if you pulled in something like absl or googletest as a subrepo, I don't think I want to run all their tests on upgrade)